lib/tasks/bower-install.js

```diff
@@ -37,32 +53,61 @@ class BowerInstallTask extends Task {
       .then(() => ui.writeLine(chalk.green('NPM: Installed bower')));
   }
 
+  get bowerJsonPath() {
+    return path.join(this.project.root, 'bower.json');
+  }
+
+  hasBowerJson() {
+    return existsSync(this.bowerJsonPath);
+  }
+
+  ensureBowerJson() {
+    if (this.hasBowerJson()) {
+      return Promise.resolve();
+    }
+
+    let projectName = this.project.name();
+    let bowerJsonPath = this.bowerJsonPath;
+
+    logger.info('Creating "bower.json" for: %s at: %s', projectName, bowerJsonPath);
+
+    return writeJson(bowerJsonPath, { name: projectName });
+  }
+
   // Options: Boolean verbose
   run(options) {
-    let bowerJson = path.join(this.project.root, '/bower.json');
     let ui = this.ui;
 
-    if (!existsSync(bowerJson)) {
-      ui.writeWarnLine('Skipping bower install: bower.json not found');
+    let packages = options.packages || [];
+    let installOptions = options.installOptions || {};
+    let savePackages = installOptions.save || installOptions.saveDev;
+
+    // if we are running "bower install" from "ember init" and there is
+    // no "bower.json" we return early
+    if (!this.hasBowerJson() && !savePackages) {
```

Comment:
Note: rather than omitting the file we can do `Object.keys().length` of `dependencies` and `devDependencies` to determine if we need to install it. It's also less likely to break `ember-try`.